control/pure_pursuit/src/pure_pursuit/pure_pursuit_lateral_controller.cpp

```diff
@@ -185,9 +185,16 @@ double PurePursuitLateralController::calcCurvature(const size_t closest_idx)
 
   if (static_cast<size_t>(closest_idx) >= idx_dist) {
     prev_idx = closest_idx - idx_dist;
+  } else {
+    // return zero curvature because prev_idx can't be found
```

Comment:
@kyoichi-sugahara
Thank you for PR.
could you tell me which case meets condition below? 
> // return zero curvature because prev_idx can't be found

> // return zero curvature because next_idx can't be found

Reply:
@taikitanaka3 
how about this?
>return zero curvature when backward/forward distance is not long enough in the trajectory